src/util/gentle-register-property.js

```diff
@@ -25,13 +25,12 @@ const INITIAL_VALUES = {
  * @param {string} [meta.syntax] - Property syntax.
  * @param {boolean} [meta.inherits] - Whether the property inherits.
  * @param {*} [meta.initialValue] - Initial value.
+ * @param {Window} [window] - The window to check in.
  */
 export default function gentleRegisterProperty (property, meta = {}, window = globalThis) {
-	let CSS = window.CSS;
-
```

Comment:
Is there a reason for getting rid of this variable?

Reply:
Not really, I can revert if you prefer. I just don't see the point of shadowing the variable, and other places that call methods on `window.CSS` don't do it.

Reply:
I have no strong opinion on this; it seems somewhat orthogonal to the main goal of the PR, so I wouldn't change files if we don't have to. It's just a preference. Let's leave it to Lea to decide. 🙂

Reply:
I think the change is fine since we only use it once, but no strong opinion either way.